Templates/QueueTrigger-FSharp/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "defaultFunctionName": "QueueTriggerFSharp",
+    "description": "An F# function that will be run whenever a message is added to a specified Azure Storage queue",
+    "name": "QueueTrigger - F#",
+    "language": "F#",
+    "trigger": "QueueTrigger",
```

Comment:
I believe the trigger metadata is being used by the CI automation, Naren can confirm. I double checked across the templates and there is some inconsistency between which ones have it and which ones dont. I will leave it here right now, we can do another pass to get this in order.